.github/workflows/tests.yml

```diff
@@ -83,11 +83,11 @@ jobs:
     strategy:
       matrix:
         config:
-          # - {device: "default.qubit", analytic: true, shots: 1000}
-          # - {device: "default.qubit", analytic: false, shots: 1000}
-          # - {device: "default.qubit.tf", analytic: true, shots: 1000}
-          - {device: "default.qubit.autograd", analytic: true, shots: 1000}
-          - {device: "default.mixed", analytic: true, shots: 1000}
+          # - {device: "default.qubit", shots: None}
+          # - {device: "default.qubit", shots: 1000}
+          # - {device: "default.qubit.tf", shots: None}

```

Comment:
I was wondering why these are commented out?

Reply:
Oh lol I commented these out. It was because the tests were taking soooo loooong to run on CI for every PR.

Do you think it's too far removed to add these devices to https://github.com/pennylaneai/plugin-test-matrix? Or should we try and run them for every PR?

Reply:
Ah no, probably better to use the test matrix then!